tests/e2e/constants/constants.go

```diff
@@ -0,0 +1,11 @@
+package constants
+
+const (
+	RunController = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/cloud/; sudo nohup ./edgecontroller > edgecontroller.log 2>&1 &"
+	RunEdgecore   = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/edge/; sudo nohup ./edge_core > edge_core.log 2>&1 &"
+	RunEdgeSite   = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/edgesite/; sudo nohup ./edgesite > edgesite.log 2>&1 &"
+
+	AppHandler        = "/api/v1/namespaces/default/pods"
+	NodeHandler       = "/api/v1/nodes"
+	DeploymentHandler = "/apis/apps/v1/namespaces/default/deployments"
+)
```

Comment:
please add a blank line at the end of this file.
Applicable to similar cases below

Reply:
done